docs/administrators/updating_the_orchestrator.rst

```diff
@@ -0,0 +1,67 @@
+.. _updating_the_orchestrator:
+
+
+Updating the orchestrator
+-------------------------
+
+Migrating from one running orchestrator to another running orchestrator
+#######################################################################
+
+This document describes the procedure to migrate from one running orchestrator
+to another one. This procedure should be followed when an in-place update of
+the orchestrator is not possible e.g. when the operating system needs to be
+updated alongside the orchestrator or when updating the orchestrator to a
+different major version.
+
+Context
++++++++
+
+The procedure below describes how to migrate from one running orchestrator
+denoted as the 'old orchestrator' to another one denoted as the 'new orchestrator'.
+
+Procedure
++++++++++
+
+
+.. tab-set::
+
+    .. tab-item:: Old orchestrator
+
+
+        1. Halt all environments (by pressing the ``STOP`` button in the dashboard for each environment).
+        2. Stop the server:
+
+        .. code-block:: bash
+
+            sudo systemctl stop inmanta-server.service
+
+        3. Make a dump of the server database using ``pgdump``.
+
+
+    .. tab-item:: New orchestrator
+
+        1. **Preliminary step**: Make sure the desired version of the orchestrator is installed, by following the
+        `installation instructions <https://docs.inmanta.com/inmanta-service-orchestrator/latest/install/1-install-server.html>`_
```

Comment:
This web-page will be shared between the documentation for OSS and several ISO versions. We should make this an internal link that references the installation documentation page instead of a fully qualified link that always goes to the OSS documentation.